golem-test-framework/src/dsl/mod.rs

```diff
@@ -1184,50 +1203,18 @@ pub fn to_worker_metadata(
     )
 }
 
-fn dump_component_info(path: &Path) -> golem_common::model::component_metadata::ComponentMetadata {
```

Comment:
This got fully moved to the file system implementation of the component service, so doing it as part of the dsl is unneeded now

Reply:
I don't think we want this dump happen in the prod service, but that can be achieved by using log levels or some configurations so I'm ok with it.

Reply:
I meant the test-framework ComponentService, not the prod one :)

https://github.com/golemcloud/golem/blob/main/golem-test-framework/src/components/component_service/mod.rs#L51